client/my-sites/hosting/sftp-card/test/index.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @jest-environment jsdom
+ */
+
+/**
+ * External dependencies
+ */
+import { shallow, mount } from 'enzyme';
```

Comment:
@glendaviesnz's comment in #38045:

> I was tempted to use '@testing-library/react' for these tests instead of enzyme, now that it is been included in the calypso project by the ajax team, but wasn't sure how keen the rest of the team was to explore this - I really like the testing philosophy behind this testing library - https://testing-library.com/

Reply:
There is already some good feedback suggesting we should move to that library too (p6fDka-Fl-p2).

However, I wonder if we should start a discussion about officially adapting this framework in Calypso before writing more tests using it. Otherwise we can end up with teams writing tests with different frameworks (and thus hard to maintain).

Reply:
That discussion doesn't look like it'll move forward any time soon 🙃

Reply:
yeh, doesn't look it is quite the time to head down the react testing library path just yet